App/src/main/res/xml/preferences.xml

```diff
@@ -0,0 +1,20 @@
+<PreferenceScreen
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+        <SwitchPreference
+            android:key="settingVisibility"
+            android:title="@string/visibility"
+            android:defaultValue="true"
+            android:summaryOn="@string/you_are_visible"
+            android:summaryOff="@string/you_are_invisible" />
```

Comment:
You can try to use the support library `SwitchCompat` as explained in http://stackoverflow.com/questions/26724566/create-own-switchcompat-preference